app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "NotVaccinatedError"
```

Comment:
The `__str__` method for `NotVaccinatedError` should return a more descriptive message. Consider returning something like 'Visitor is not vaccinated.' to provide clarity when the error is raised.